javascript-loops-practice/add-suffix-to-all.js

```diff
@@ -0,0 +1,8 @@
+/* exported addSuffixToAll */
+function addSuffixToAll(words, suffix) {
+  for (var i = 0; i < words.length; i++) {
+    words[i] += suffix;
```

Comment:
This is a no-no. When passed arrays or objects as arguments, you should _**never**_ change the original, unless that was the specified intent. This function asks for a "new" array to be returned in the instructions, you should be making a copy of the original values with the suffix added, and leave the original array from the argument unchanged